@kiva/kv-components/vue/KvLoanActivities.vue

```diff
@@ -0,0 +1,253 @@
+						{{ formattedDate(activity.key) }}
+					</h4>
+					<div class="tw-flex tw-flex-col tw-gap-y-1">
+						<KvActivityRow
+							v-for="(act, index) in activity.data"
+							:key="index"
+							:activity="act"
+						/>
+					</div>
+				</div>
+			</div>
+			<div>
+				<p
+					v-if="errorMsg"
+					class="tw-mt-1 tw-text-small tw-text-danger"
+				>
+					{{ errorMsg }}
+				</p>
+				<div class="tw-flex tw-justify-end tw-mt-4 tw-pb-2.5 md:tw-pb-4">
+					<KvLendCta
+						:loan="loan"
+						:is-loading="false"
+						:kv-track-function="kvTrackFunction"
+						:get-cookie="getCookie"
+						:set-cookie="setCookie"
+						:user-balance="userBalance"
+						:basket-items="basketItems"
+						:route="route"
+						:is-adding="isAdding"
+						:enable-five-dollars-notes="enableFiveDollarsNotes"
+						@add-to-basket="$emit('add-to-basket', $event)"
+					/>
+				</div>
+			</div>
+		</KvLightbox>
+	</div>
+</template>
+
+<script>
+import { computed, ref, toRefs } from 'vue-demi';
+import numeral from 'numeral';
+import { format } from 'date-fns';
+import KvActivityRow from './KvActivityRow.vue';
+import KvLightbox from './KvLightbox.vue';
+import KvLendCta from './KvLendCta.vue';
+
+export default {
+	name: 'KvLoanActivities',
+	components: {
+		KvActivityRow,
+		KvLightbox,
+		KvLendCta,
+	},
+	props: {
+		/**
+		 * loan object coming from parent component
+		 */
+		loan: {
+			type: Object,
+			default: () => ({}),
+		},
+		/**
+		 * loan combined activities coming from parent component
+		 */
+		combinedActivities: {
+			type: Array,
+			default: () => ([]),
+		},
+		/**
+		 * isAdding flag coming from parent add to basket method
+		 */
+		isAdding: {
+			type: Boolean,
+			default: false,
+		},
+		/**
+		 * basketItems array coming from parent component
+		 */
+		basketItems: {
+			type: Array,
+			default: () => ([]),
+		},
+		/**
+		 * userBalance string coming from parent component
+		 */
+		userBalance: {
+			type: String,
+			default: undefined,
+		},
+		/**
+		 * errorMsg string coming from parent add to basket method
+		 */
+		errorMsg: {
+			type: String,
+			default: undefined,
+		},
+		/**
+		 * kvTrackFunction function to track events from LendCta component
+		 */
+		kvTrackFunction: {
+			type: Function,
+			required: true,
+		},
+		/**
+		 * getCookie function for LendCta component
+		 */
+		getCookie: {
+			type: Function,
+			default: undefined,
+		},
+		/**
+		 * setCookie function for LendCta component
+		 */
+		setCookie: {
+			type: Function,
+			default: undefined,
+		},
+		/**
+		 * route object for LendCta component
+		 */
+		route: {
+			type: Object,
+			default: undefined,
+		},
+		enableFiveDollarsNotes: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	emits: [
+		'add-to-basket',
+	],
+	setup(props) {
+		const {
+			loan,
+			combinedActivities,
+			kvTrackFunction,
+		} = toRefs(props);
+
+		const lightboxOpen = ref(false);
+
+		const modalTitle = computed(() => `Activity for ${loan.value?.name}`);
```

Comment:
Is it possible to add `data-hj-suppress` to the modal title for the borrower name?

Reply:
We can do that adding a title slot to the KvLightbox used here...